roles/pgbackrest_srv/tasks/main.yml

```diff
@@ -0,0 +1,115 @@
+---
```

Comment:
This code duplicates roles/pgbackrest/tasks/main.yml

I think it should be one pgbackrest role and not two roles with the same code